src/main/java/uk/gov/digital/ho/hocs/casework/api/CaseNoteResource.java

```diff
@@ -46,4 +46,18 @@ public CaseNoteResource(CaseNoteService caseNoteService) {
         CaseNote caseNote= caseNoteService.createCaseNote(caseUUID, createCaseNoteRequest.getType(), createCaseNoteRequest.getText());
         return ResponseEntity.ok(caseNote.getUuid());
     }
+
+    @Authorised(accessLevel = AccessLevel.OWNER)
```

Comment:
Shouldn't the access level be the same as the one for createCaseNote()?

Reply:
I down graded the Create endpoint so that the Task Force team can add notes.
The Task Force team should not be allowed to edit/delete case notes - that'll be more a manager role.